Products/product.css

```diff
@@ -28,11 +28,24 @@
 
 .ecom-card {
     width: 25rem;
-    height: 45rem;
+    height: 47rem;
+}
+
+.card-btn {
+    width: 90%;
+}
+
+.action-btns {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
This can be extracted into a utils class

Reply:
Already have an util class but isn't working. Let me Recheck.